packages/framework/src/Resources/views/Admin/Form/productParameterValue.html.twig

```diff
@@ -25,7 +25,7 @@
 
 {% block product_parameter_value_widget %}
     {% import _self as self %}
-    <div class="js-form-group wrap-border form-group">
+    <div class="js-form-group form-group">
```

Comment:
here was removed `wrap-border` but no `border-divider` div element was added, is this intentional?